doc/tutorial/introduction/introduction_read_fasta.cpp

```diff
@@ -1,16 +1,18 @@
 //! [read]
-#include <string>                            // for std::string
-#include <vector>                            // for std::vector
-#include <seqan3/argument_parser/all.hpp>    // for argument_parser
-#include <seqan3/core/debug_stream.hpp>      // for debug_stream
-#include <seqan3/io/sequence_file/input.hpp> // for sequence_file_input
+#include <string>                               // for std::string
+#include <vector>                               // for std::vector
+#include <seqan3/alphabet/nucleotide/dna5.hpp>  // for dna5 datastrucutre
```

Comment:
sorry, found one more thig: separate stl headers from seqan3 headers

Reply:
I amended the last commit.